### ISSUE TITLE
Add support for groups

### ISSUE THREAD

**Issue by SkypLabs** [ente-io/auth#8]
Hi,

I'd like to suggest adding groups to easily display only personal or work items for example. You can have a look at how Aegis does it for inspiration.

And thanks for this great tool!

Cheers

**Comment by michaelsara** [comment]
I agree that grouping is too easy to use for a large number of scenarios, and it is highly recommended that developers consider this feature
Also thanks guys,  for this great tool!

**Comment by 0uros** [comment]
I would love for this to be added soon! It's something that has 2FAS Auth, and I love it
